sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/specialized/AppendBlobAsyncClient.java

```diff
@@ -152,7 +153,8 @@
     public Mono<Response<AppendBlobItem>> createWithResponse(BlobHttpHeaders headers, Map<String, String> metadata,
         BlobRequestConditions requestConditions) {
         try {
-            return withContext(context -> createWithResponse(headers, metadata, requestConditions, context));
+            return withContext(context -> createWithResponse(headers, metadata, requestConditions, context),
+                STORAGE_TRACING_PROPERTIES);
```

Comment:
Since it seems like pretty much every call to `withContext` needs to pass this constant, I think it'd be helpful if we defined something in storage.common that's like withTracingContext or something to automatically add this. Otherwise, I think us Storage folks are liable to forget and mess up tracing later :)

Reply:
Updated with a `withTracingContext` in Storage Common that will update the returned serviceCall context with `STORAGE_TRACING_PROPERTIES`.